test/skate_web/controllers/detours_controller_test.exs

```diff
@@ -58,6 +58,59 @@ defmodule SkateWeb.DetoursControllerTest do
     end
   end
 
+  describe "get_detours/2" do
+    @tag :authenticated
+    test "fetches detours from database and groups by active, past, draft", %{conn: conn} do
```

Comment:
I think it would be worthwhile to add test cases that validate:
1. that a user cannot retrieve a different user's drafts, and
2. that the endpoint will return the properly-formatted detours, even if there are some detours that are not properly formatted.

Reply:
1. User cannot retrieve a different user's drafts: isn't that covered by [this comment](https://github.com/mbta/skate/pull/2757#discussion_r1742458418) which you re-reviewed given [this comment](https://github.com/mbta/skate/pull/2757#discussion_r1745956857)?
2. This is new and can do.

Reply:
1. That asserts that the author ID comes back correctly, but there's no test asserting that if I create a draft and then you load your drafts, that my draft won't show up in that list.
2. 🎉